conf/defaults.ini

```diff
@@ -230,6 +230,12 @@ snapshot_remove_expired = true
 # Number dashboard versions to keep (per dashboard). Default: 20, Minimum: 1
 versions_to_keep = 20
 
+#################################### Annotations ##################
+
+[annotations]
+# Number of days to keep annotations (per alert). Default: 60, Minimum: 5
```

Comment:
This comment says per alert but the SQL statement deletes all kinds of annotations. I think we should be clear about this and perhaps have different settings per kind of annotation.

Reply:
@bergquist Thanks for clarifying that :+1: